include/swift/AST/DiagnosticsSema.def

```diff
@@ -825,6 +825,10 @@ ERROR(argument_out_of_order_unnamed_unnamed,none,
       "unnamed argument #%0 must precede unnamed argument #%1",
       (unsigned, unsigned))
 
+ERROR(member_shadows_global_function,none,
+      "member '%0.%2' shadows global function '%1.%2'",
```

Comment:
@jrose-apple thanks for the pointer, using DescriptiveDeclKind would make the message much better!